### PR TITLE
Fixing DeferredManager.StartPolicy.MANAUL Bad spelling

### PR DESCRIPTION
**Pull request by AbdelRahmaneB**
Fixes for #52

### CODE REVIEW

**Comment by aalmiray** [comment]
This introduces a binary incompatibility in the next release. We have the following options:

* keep the typo. fix it in version 2.x
* add another entry to the enum, making sure its semantic value is equal to the entry with the typo

option #1 is the safest as you can't ensure everyone will switch to the new enum entry. wdyt @saturnism?

**Comment by AbdelRahmaneB** [comment]
Yes , why not the two options ? :)
Deprecate the typo, and remove it in 2.x  !? 
Something like that : 

```
public static enum StartPolicy {
		/**
		 * Let Deferred Manager to determine whether to start the task at its own
		 * discretion.
		 */
		DEFAULT,
		
		/**
		 * Tells Deferred Manager to automatically start the task
		 */
		AUTO,
		
		/**
		 * Tells Deferred Manager that this task will be manually started
		 *
         * @deprecated  As of Version 1.x, this element is deprecated.
		 *		Use MANUAL instead. 
		 * 		It will be removed in version 2.x
		 */
        @Deprecated
		MANAUL

		/**
		 * Tells Deferred Manager that this task will be manually started
		 */
		MANUAL
	}
```

**Comment by saturnism** [comment]
I generally like deprecation first to have a smoother transition to newer versions, but it will require more test cases to be written, and will need to be thrown away by v2.

semver prefers deprecation firs too. we can do that if there is bandwidth to get the code updates & test cases in :)

**Comment by aalmiray** [comment]
Based on @saturnism's last comment, @AbdelRahmaneB could you update the PR with a deprecate warning on the enum constant? We'd also need a section in the README that mentions deprecations IMHO.

**Comment by AbdelRahmaneB** [comment]
Done, thanks @aalmiray . About depracations section , is it better to add it to a wiki page?

**Comment by saturnism** [comment]
We don't have wikipage yet. README should be fine for now. Thanks!

**Comment by AbdelRahmaneB** [comment]
@saturnism  I've added a README deprecations section.

File: README.md [hunk 1]
```diff
@@ -387,3 +387,10 @@ var google_remarketing_only = false;
 <img height="1" width="1" style="border-style:none;" alt="" src="//www.googleadservices.com/pagead/conversion/974052972/?value=0&amp;label=wsVZCOycvgkQ7Ly70AM&amp;guid=ON&amp;script=0"/>
 </div>
 </noscript>
+
+<a name="deprecations"></a>Depracations
+==============
+
+<a name="deprecations-v1.2.5"></a>v1.2.5
+--------
+* ~~```DeferredManager.StartPolicy.MANAUL```~~ is deprecated and will be removed in the next major version. Use ```DeferredManager.StartPolicy.MANUAL``` instead.
```
Comment (saturnism) [inline README.md]: "will be removed in the next minor version" not major, in this case. We'll remove it in 1.3.

Comment (AbdelRahmaneB) [inline README.md]: Ok, so the deprecation warning on the enum should also be changed.

**Comment by saturnism** [comment]
lgtm, i feel it's ready to be merged.

**Pull request merged**